libs/template/presets/src/b2c/app.ts

```diff
@@ -39,6 +40,12 @@ const isPreview = new URLSearchParams(
 export const b2cResources: Resources = {
   graphics: commonGraphics,
   icons: storefrontIcons,
+  fonts: {
+    [materialMapper.id]:
+      'https://fonts.googleapis.com/icon?family=Material+Icons',
```

Comment:
```suggestion
    [materialMapper.id]:
      'https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200',
```

We like to use material variable font icons, so we can adjust fill and line weight by css.